src/devices/third_reality.ts

```diff
@@ -406,6 +406,74 @@ const definitions: DefinitionWithExtend[] = [
             }),
         ],
     },
+    {
+        zigbeeModel: ['3RDP01072Z'],
+        model: '3RDP01072Z',
+        vendor: 'Third Reality',
+        description: 'Zigbee / BLE dual plug with power',
```

Comment:
Can you remove everything below this line and try with:

```
ota: true
extend: [deviceEndpoints({endpoints: {left: 1, right: 2}}), onOff({endpointNames: ['left', 'right']}), electricityMeter({endpointNames: ['left', 'right']})],
```

Reply:
ok

Reply:
Can you now try after removing `fromZigbee`, `toZigbee` and `exposes`?

Reply:
ok